core/src/wallet/algorand/AlgorandAccount.cpp

```diff
@@ -94,26 +96,36 @@ namespace algorand {
 
     bool Account::putBlock(soci::session& sql, const api::Block& block)
     {
-        if (BlockDatabaseHelper::putBlock(sql, block)) {
```

Comment:
Where is `BlockDatabaseHelper::putBlock` called with this new implementation?

Reply:
1- in algorand::OperationsDatabaseHelper::bulkInsert for operations block
2- I did a rollback because of the last block query (when Account::putBlock is called by the synchronizer)